tsconfig.types.json

```diff
@@ -6,8 +6,9 @@
     "skipLibCheck": true,
     "preserveConstEnums": true,
     "outDir": "dist",
-    "rootDir": "./src"
+    "rootDir": ".",
```

Comment:
I had to change the `rootDir` in order to include an import from the public folder. Might be worth changing back again once we are converting the Icon component and (maybe) change the way we are mapping the icon codes.